crates/bevy_ui/src/ui_node.rs

```diff
@@ -1832,14 +1839,43 @@ pub struct UiImage {
     pub flip_y: bool,
 }
 
+impl Default for UiImage {
+    /// A solid square, with a transparent white color.
+    fn default() -> Self {
+        UiImage {
+            // This needs to be transparent by default, to avoid covering the background color
+            // This should be white because the tint is multiplied with the image,
+            // so if you set an actual image with default tint you'd want its original colors
+            color: Color::from(LinearRgba::new(1.0, 1.0, 1.0, 0.0)),
```

Comment:
Isn't the Color default already pure white? :thinking:

Reply:
It's solid white, not transparent white.

Reply:
Oh yeah, My bad.